Package:     fox1.6
Subject:     ld with --no-add-needed
Author:      Matthias Klose <doko at ubuntu.com>
Forwarded:   no
Last-Update: 2012-05-13

Fix build failure with ld --no-add-needed.
---

diff -urN s02/tests/Makefile.am s03/tests/Makefile.am
--- s02/tests/Makefile.am	2012-05-13 10:27:30.791941110 +0200
+++ s03/tests/Makefile.am	2012-05-13 10:27:40.443411102 +0200
@@ -4,7 +4,7 @@
 INCLUDES = -I$(top_srcdir)/include -I$(top_builddir)/include
 RESWRAP  = $(top_builddir)/utils/reswrap
 SUFFIXES = .gif .bmp
-LDADD = $(top_builddir)/src/libFOX-1.6.la -lm
+LDADD = $(top_builddir)/src/libFOX-1.6.la -lm -lX11
 
 noinst_PROGRAMS	= \
 expression \
